hmac/decryptor/hashQuery_test.go

```diff
@@ -64,36 +63,48 @@ func TestSearchablePreparedStatementsWithTextFormat(t *testing.T) {
 	}), mock.Anything).Return(nil)
 	_ = bindValue
 
-	queryObj := base.NewOnQueryObjectFromQuery(query, parser)
-	queryObj, _, err = encryptor.OnQuery(ctx, queryObj)
-	if err != nil {
-		t.Fatal(err)
-	}
-	bindPlaceholders := encryptor2.PlaceholderSettingsFromClientSession(clientSession)
-	if len(bindPlaceholders) != 1 {
-		t.Fatal("Not found expected amount of placeholders")
-	}
-	queryObj = base.NewOnQueryObjectFromQuery(query, parser)
-	statement, err := queryObj.Statement()
-	if err != nil {
-		t.Fatal(err)
+	type testcase struct {
+		Query string
 	}
-	newVals, ok, err := encryptor.OnBind(ctx, statement, []base.BoundValue{boundValue})
-	if err != nil {
-		t.Fatal(err)
-	}
-	if !ok {
-		t.Fatal("Values should be changed")
-	}
-	if len(newVals) != 1 {
-		t.Fatal("Invalid amount of bound values")
-	}
-	setting := schema.GetTableSchema("test_table").GetColumnEncryptionSettings("data1")
-	newData, err := newVals[0].GetData(setting)
-	if err != nil {
-		t.Fatal(err)
+	testcases := []testcase{
+		{Query: "SELECT data1 from test_table WHERE data1=$1"},
```

Comment:
can you add unit-tests for these cases but with value literals? to test it without prepared statements too.